webrender/src/frame.rs

```diff
@@ -320,6 +320,10 @@ impl Frame {
                                                   background_color,
                                                   self.frame_builder_config);
 
+        if let Some(ref mut old_builder) = self.frame_builder {
```

Comment:
I think it would be cleaner to do something like
```rust
let mut frame_builder = self.frame_builder.take().unwrap_or(FrameBuilder::new(...));
frame_builder.clear(); // resets all the contents, keeps the capacity/allocation
```